Kudu.Core/Infrastructure/WindowsLockFile.cs

```diff
@@ -310,6 +311,20 @@ private void TraceIfUnknown(Exception ex)
             }
         }
 
+        private void IsLockFileRemoved()
```

Comment:
this method name implies it is a function which returns a boolean, but the actual code doesn't match the method name expectation.